### PR TITLE
Patch 1 (#2)

### PR DESCRIPTION
**Pull request by Sliqric7053**
* Fix typo

Changed "RESTROED" to "RESTORED"

* update typo

### CODE REVIEW

File: Comanche055/JET_SELECTION_LOGIC.s [hunk 1]
```diff
@@ -277,7 +277,7 @@ BDZCHECK	CA	ZNDX
 		EXTEND
 		BZF	TAUCHECK
 		
-ACRBDZ		CA	T5TEMP		# Z-TRANSLATION ACCEPTED EVEN THO WE MAY
+ACRBDZ		CA	T5TEMP		# Z-TRANSLATION ACCEPTED EVEN THOUGH WE MAY
```
Comment (oldmud0) [inline Comanche055/JET_SELECTION_LOGIC.s]: [Rejected](https://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1044.jpg)

File: Comanche055/UPDATE_PROGRAM.s [hunk 2]
```diff
@@ -59,7 +59,7 @@
 # RESTARTS:		P27 IS RESTART PROTECTED IN TWO WAYS ...
 #			1.	PRIOR TO VERIFLAG INVERSION (WHICH IS CAUSED BY THE GROUND/ASTRONAUT'S VERIFICATION OF UPDATE
 #				DATA BY SENDING A V33E WHEN V21N02 IS FLASHING)---
-#				NO PROTECTION EXCEPT PRE-P27 MODE IS RESTROED, COAST + ALIGN DOWNLIST IS SELECTED AND UPLINE
+#				NO PROTECTION EXCEPT PRE-P27 MODE IS RESTORED, COAST + ALIGN DOWNLIST IS SELECTED AND UPLINE
```
Comment (oldmud0) [inline Comanche055/UPDATE_PROGRAM.s]: [Accepted](https://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1497.jpg)

**Comment by oldmud0** [comment]
Duplicate of #132.

**Comment by wopian** [comment]
@Sliqric7053 opened this new PR as advised by Chris in that PR to solve the merge conflicts.

https://github.com/chrislgarry/Apollo-11/pull/132#issuecomment-238007611

>Or, if you used the web interface and not the command line, you can just close this and go do the same thing you did before which will open a new request without the conflicts.

**Comment by oldmud0** [comment]
First, this one has merge conflicts too. Second, he squashed two changes, one of which is rejected.

**Pull request closed without merging**